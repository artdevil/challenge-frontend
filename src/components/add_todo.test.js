import React from 'react';
import { shallow } from 'enzyme';

import AddTodo from './add_todo';

describe('<AddTodo />', () => {
  it('shallow add_todo renders without crashing', () => {
    const wrapper = shallow(<AddTodo />);
    expect(wrapper).toMatchSnapshot();
  });
});
