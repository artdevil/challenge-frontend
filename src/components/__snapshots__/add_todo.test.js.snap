// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AddTodo /> shallow add_todo renders without crashing 1`] = `
<AddTodo
  mutate={[Function]}
/>
`;
